hastings-center-report.csl

```diff
@@ -0,0 +1,1259 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" et-al-min="4" et-al-use-first="1" initialize-with=". " page-range-format="chicago" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Hastings Center Report</title>
+    <title-short>HCR</title-short>
+    <id>http://www.zotero.org/styles/hastings-center-report</id>
```

Comment:
Usually we go by the name on the cover, which would be "The Hastings Center Report". Would that be acceptable? This would mean updating the style file name, ID, "self" link, and title.